inclusive-design/wecount.inclusivedesign.ca
# feat: SVG logo

## Description

This PR adds an SVG logo which can be styled using the UIO/UIO+ contrast modes feature.

## Steps to test

1. Open the deploy preview in Chrome.
2. Test all contrast modes using UIO+ extension.

**Expected behavior:** Logo changes as expected to match colour schemes.

## Additional information

Not applicable.

## Related issues

Not applicable.
